### PR TITLE
Enable testing for smart_proxy_pulp

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/plugins/smart_proxy_pulp.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+- project:
+    name: smart_proxy_pulp
+    defaults: proxy_plugin
+    min_ruby193: true
```
Comment (domcleal) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/plugins/smart_proxy_pulp.yaml]: Is this correct? The repository seems to suggest it supports Ruby 1.8.7.

File: puppet/modules/slave/templates/test_pull_requests_smart_proxy_pulp.json.erb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            "branches": {
+                "*": {
+                    "jenkins_job_name": "test_proxy_plugin_pull_request",
+                    "downstream_job_name": "test_proxy_plugin_smart_proxy_pulp_master",
+                    "build_token": "<%= jenkins_build_token -%>",
+                    "addtl_jenkins_params": {
+                        "smart_proxy_pulp": {
+                            "plugin_repo": "https://github.com/theforeman/smart_proxy_pulp.git",
+                            "plugin_name": "smart_proxy_pulp",
+                            "min_ruby193": "true"
```
Comment (domcleal) [inline puppet/modules/slave/templates/test_pull_requests_smart_proxy_pulp.json.erb]: ditto

**Comment by stbenjam** [comment]
Fixed and thanks

File: puppet/modules/slave/templates/test_pull_requests_smart_proxy_pulp.json.erb [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            "run_without_trigger": true,
+            "branches": {
+                "*": {
+                    "jenkins_job_name": "test_proxy_plugin_pull_request",
+                    "downstream_job_name": "test_proxy_plugin_smart_proxy_pulp_master",
+                    "build_token": "<%= jenkins_build_token -%>",
+                    "addtl_jenkins_params": {
+                        "smart_proxy_pulp": {
+                            "plugin_repo": "https://github.com/theforeman/smart_proxy_pulp.git",
+                            "plugin_name": "smart_proxy_pulp",
```
Comment (domcleal) [inline puppet/modules/slave/templates/test_pull_requests_smart_proxy_pulp.json.erb]: Invalid JSON syntax, the trailing comma must be removed

**Comment by stbenjam** [comment]
Sorry, fixed.  It's green.

**Comment by domcleal** [comment]
Merged, thanks @stbenjam.

**Pull request merged**